Resources/Public/JavaScript/find.js

```diff
@@ -267,19 +267,19 @@ var tx_find = (function () {
      *
      * @param {object} ranges
      */
-    var selectRanges = function (ranges) {
+    var selectRanges = function (event, ranges) {
       var newRange = roundedRange(ranges.xaxis);
       plot.setSelection({'xaxis': newRange}, true);
       hideTooltip();
-      startSearchWithNewFacet(newRange);
+      startSearchWithNewFacet(event, newRange);
     };
 
     jGraphDiv.bind('plotclick', function (event, pos, item) {
       return true;
     });
 
-    jGraphDiv.bind('plotselected', function (event, ranges) {
-      selectRanges(ranges);
+    jGraphDiv.one('plotselected', function (event, ranges) {
+      selectRanges(event, ranges);
```

Comment:
As far as I can see, the only property of the `event` that is used is the `event.target` in a function above. Should we reduce the method argument to only pass / take `event.target` instead of the complete `event` object?

Reply:
¯\_(ツ)_/¯

To me, passing the whole (well established) event object still feels quite natural for jQuery code, even some years later.

I see the point for custom objects, though.